styles/base/forms.css

```diff
@@ -0,0 +1,18 @@
+fieldset {
+  border: 0;
+  margin: 0;
+  padding: 0;
+}
+
+/* Chrome, Safari, Edge, Opera */
+input::-webkit-outer-spin-button,
+input::-webkit-inner-spin-button {
+  appearance: none;
+  margin: 0;
+}
```

Comment:
![Screenshot 2024-01-15 at 13 56 30](https://github.com/etchteam/diamond-ui/assets/5038459/4a461ce3-dd86-42a1-9408-ddbf659923ac)

For this file – Would you expect installing Diamond to make these browser alterations to any website that installs it. Or is Diamond intended to be just for our projects and opinions?